packages/core-sdk/test/unit/resources/group.test.ts

```diff
@@ -553,4 +553,136 @@ describe("Test IpAssetClient", () => {
       expect(result.encodedTxData!.data).to.be.a("string").and.not.empty;
     });
   });
+
+  describe("Test groupClient.collectAndDistributeGroupRoyalties", async () => {
+    it("should throw error when call collectAndDistributeGroupRoyalties given group ip id is not registered", async () => {
```

Comment:
lets try to keep the test name a bit more concise for improved readability. I feel like our test name are getting pretty long without conveying too much.
For example, I think this can be `it("throws error if group ipId is not registered")`